pkg/tasks/prometheus.go

```diff
@@ -230,6 +232,40 @@ func (t *PrometheusTask) Run() error {
 		return errors.Wrap(err, "reconciling Prometheus kube-controllers ServiceMonitor failed")
 	}
 
+	if t.config.EtcdConfig != nil && t.config.EtcdConfig.Enabled != nil && *t.config.EtcdConfig.Enabled {
```

Comment:
General question: Why do we have the `Enabled` property? Why don't we see etcd as enabled when the `EtcdConfig != nil`?

Reply:
You're right let's stay more consistent with the rest of our components.